lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc

```diff
@@ -136,3 +142,52 @@ Students who  can toggle between verbal expressions, Circles of Evaluation, and
 - How do you decide the order in which to apply absolute value and opposite when an expression includes both?
 
 - Did you work from the inside-out or the outside-in when evaluating Circles of Evaluation with absolute value and opposite? Did your strategy change, depending on the Circle of Evaluation? Explain.
+
+
+== Programming Exploration
+
+=== Overview
+
+Students apply their knowledge of examples in @proglang to think about @show{(code 'abs)} and `negate`.
+
+=== Launch
+
+We are going to complete an activity that involves (1) making predictions about equations with absolute value and negation, and (2) running tests in @proglang to see if our predictions are correct, and then (3) reflecting on what we learned.
+
+@lesson-instruction{
+On @printable-exercise{examples.adoc}, complete the table at the top of the page. If you get stuck, translate the code into Circles of Evaluation. Discuss your predictions with your partner.
+}
+
+@teacher{Students may need you to demonstrate your thought process for one or two of the examples.
+
+When finished, invite students to share their predictions.}
+
+@slidebreak
+
+=== Investigate
+
+@lesson-instruction{
+- With your partner, complete the second section of @printable-exercise{examples.adoc}.
+- If you discover that one of your predictions was wrong, revise the table at the top of the page.
+}
+
+@teacher{
+As you circulate, ensure that students are looking at the messages that appear in @proglang. This activity not only provides practice thinking about the absolute value and opposite; it also gives students exposure to tests - bits of code used to verify that code is working as we would expect. Examples and tests are widely used in programming! We explore examples in greater depth in @lesson-link{functions-examples-definitions}.}
+
+@slidebreak
+
+@lesson-instruction{
+- Complete the final section of @printable-exercise{examples.adoc}.
+- How can you use @proglang to help you decide if an equation is true?
+}
+
+Two algebraic expressions are @vocab{equivalent} if they produce the same outcome, no matter what value we substitute in for the variable(s). In this activity, *none* of the equations that tested *always* passed... which means that none of them were true.
+
+@QandA{
+Can you write an algebraic equation that includes both @show{(code 'abs)} and `negate` which is *always* true?
```

Comment:
Conventional algebra doesn't have `negate` as a function. I'm guessing you mean "can you write an example/equation in Pyret.."?